nengo/tests/conftest.py

```diff
@@ -119,6 +120,15 @@ def analytics(request):
     return analytics.__enter__()
 
 
+@pytest.fixture
+def analytics_data(request):
+    paths = request.config.getvalue('compare')
+    function_name = parametrize_function_name(request, re.sub(
+        '^test_[a-zA-Z0-9]*_', 'test_', request.function.__name__, count=1))
```

Comment:
If we always name our tests `test_compare_*`, then this will always be replacing `test_compare_` with `test_`, right? Maybe it's better to keep it general like this, but it took me a while to figure out what this was doing.